index.bs

```diff
@@ -188,15 +188,17 @@ values for each, as they are all values constructed at the [=user agent=]'s whim
 
 [=User agents=] MUST map higher-entropy [=platform architecture=] values to the following buckets:
 
-* x86 CPU architectures => "x86"
-* ARM CPU architectures => "arm"
+* 32 bit x86 CPU architectures => "x86"
+* 64 bit x86 CPU architectures => "x86_64"
+* 32 bit ARM CPU architectures => "arm"
+* 64 bit ARM CPU architectures => "arm_64"
 
 Other CPU architectures could be mapped into one of these values in case that makes sense, or be
 mapped to the empty string.
 
 ISSUE 105: There might be use-cases for higher-entropy and more specific CPU architectures (e.g.
-"bitness" or for the download of highly optimized executable binaries). If necessarrry, we could
-support those use-cases through one or more separate hints.
+for the download of highly optimized executable binaries). If necessarrry, we could support those
```

Comment:
Typo, necessarrry.
(I think your 'r' key may be misbehaving, it's a second of these I noticed..)

Reply:
indeed... (as well as me 'e', and occasionally my 't'). COVID-19 is holding up refreshes...